lab-anthony/server.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const debug = require('debug')('drink:server');
+const express = require('express');
+const morgan = require('morgan');
+
+const drinkRouter = require('./route/drink-router');
+const sendError = require('./lib/error-response');
+
+const app = express();
+
+const port = process.env.PORT || 3000;
+
+app.use(morgan('dev'));
+app.use(sendError);
+app.use('/api/drink', drinkRouter);
+
+app.all('*', function(req, res){
+  debug('* 404');
+  res.status(404).send('not found');
+});
+
+const server = app.listen(port, function(){
+  debug('listen');
+  console.log('server started on port', port);
+});
+
+server.isRunning = true;
```

Comment:
This should go in the callback so that it's only called when you know the server actually is running.